_data/openseeds/ols-8/schedule.yaml

```diff
@@ -148,6 +148,16 @@ weeks:
       notes: https://bit.ly/ols-8-week-06
       title: Open Science Garden I
       type: Cohort
+      before: '- Look up two other projects and comment on their issues with feedback
```

Comment:
Could we revert the change on the content here?

Reply:
I have reverted this change. 🙏🏽